### PR TITLE
Docs Content: Toast

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
Updates content for Toast documentation.

### CODE REVIEW

**Comment by joejezowski-okta** [comment]
@edburyenegren-okta @arnoldsandoval-okta 

Can we make the buttons to see the live toast Primary buttons? I think that is proper usage here…

Also in light of the Space theme…should we change the CTA "Launch Toast" ?

File: packages/docs/components/toast.md [hunk 1]
```diff
@@ -158,13 +145,13 @@ When using a Caution Toast, be sure the user does not require more context or gu
     <OdsToast
       class="is-sample-animation-off"
       variant="caution"
-      title="This is a caution toast"
+      title="Thirty minutes to launch"
```
Comment (arnoldsandoval-okta) [inline packages/docs/components/toast.md:148]:
```suggestion
      title="T-minus one hour and thirty minutes"
```

File: packages/docs/components/toast.md [hunk 2]
```diff
@@ -158,13 +145,13 @@ When using a Caution Toast, be sure the user does not require more context or gu
...
     />
     <OdsToast
       class="is-sample-animation-off"
       variant="caution"
-      title="This is a caution toast"
-      body="It informs the user about a crucial decision-point."
+      title="Thirty minutes to launch"
```
Comment (arnoldsandoval-okta) [inline packages/docs/components/toast.md:153]:
```suggestion
      title="T-minus one hour and thirty minutes"
```

**Review by arnoldsandoval-okta — APPROVED** [review]
All nits.

**Comment by edburyenegren-okta** [comment]
@arnoldsandoval-okta @joejezowski-okta thanks for the feedback! included all changes with some wording swaps for best visibility on page

**Pull request merged**